fbgemm_gpu/codegen/training/backward/embedding_backward_dense_host_cpu.cpp

```diff
@@ -171,7 +171,8 @@ Tensor split_embedding_codegen_lookup_dense_function(
         Tensor>& /* vbe_B_offsets_rank_per_feature = std::nullopt */,
     c10::SymInt /* max_B = -1 */,
     c10::SymInt /* max_B_feature_rank = -1 */,
-    c10::SymInt /* vbe_output_size = -1 */) {
+    c10::SymInt /* vbe_output_size = -1 */,
+    bool /* mixed_D = false */) {
```

Comment:
If https://github.com/pytorch/FBGEMM/pull/3488#discussion_r1881738021 applied, consider fixing the comment